src/proteus/dswx_hls.py

```diff
@@ -2497,8 +2586,31 @@ def parse_runconfig_file(user_runconfig_file = None, args = None):
 
         setattr(args, args_name, runconfig_layer_file)
 
+    # Browse Image Filename
+    if browse_image_group['save_browse']:
+        # Get user's CLI input for the browse image filename
+        cli_arg_name = 'output_browse_image'
+        cli_browse_fname = getattr(args, cli_arg_name)
+
+        # Construct the default browse image filename per the runconfig
+        product_basename = (f'{product_id}_v{PRODUCT_VERSION}.png')
+        default_browse_fname = os.path.join(output_directory,
+                                            product_basename)
+
+        # If a browse image filename was provided via CLI, it takes
+        # precendence over the default filename.
+        if cli_browse_fname is not None:
+                logger.warning(f'command line {cli_arg_name} "{cli_browse_fname}" has'
+                                f' precedence over default {cli_arg_name}'
+                                f' "{default_browse_fname}".')
+                # `args` already contains the correct filename; no need to update.
```

Comment:
```suggestion
            logger.warning(f'command line {cli_arg_name} "{cli_browse_fname}" has'
                            f' precedence over default {cli_arg_name}'
                            f' "{default_browse_fname}".')
            # `args` already contains the correct filename; no need to update.
```

Reply:
The indentation of this block seems to be a bit messed up. Please check that in the text editor.

Reply:
Fixed.